src/emulator/functionsEmulatorRuntime.ts

```diff
@@ -401,14 +408,6 @@ function InitializeNetworkFiltering(frb: FunctionsRuntimeBundle): void {
         return original(...args);
       } catch (e) {
         const newed = new original(...args);
-        if (bundle.name === "google-gax") {
```

Comment:
We no longer need this bit since we now require `firebase-admin` v8+ and therefore we have a version of Firestore where we can pass the `Bearer owner` header in through the settings constructor.  So this was just unnecessary complexity.